### PR TITLE
(#25) testing

### PR DESCRIPTION
**Pull request by rathorevaibhav**
PR #25

### CODE REVIEW

**Comment by rathorevaibhav** [comment]
@nishanth-kd Travis has been integrated. Right now, we just have a sample unit test case. I think we can add unit test cases in separate trackings. TravisCI is complete.

Travis CI URL:
https://travis-ci.org/ColoredCow/employee-portal

**Comment by kd-csp** [comment]
@rathorevaibhav :+1:

File: .env.travis [hunk 1]
```diff
@@ -0,0 +1,11 @@
+APP_ENV=testing
+APP_KEY=SomeRandomString
```
Comment (kd-csp) [inline .env.travis]: not the expert but is it supposed to be this?

File: tests/Unit/UserTest.php [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+class UserTest extends TestCase
+{
+    /**
+     * A basic test example.
+     *
+     * @return void
+     */
+    public function testExample()
+    {
+        $this->assertTrue(true);
```
Comment (kd-csp) [inline tests/Unit/UserTest.php]: 👍

File: .env.travis (hunk 1 above)
Comment (rathorevaibhav) [reply .env.travis]: We can leave it empty. The travis.yml contains `php artisan key:generate` so, we can make it just like .env.example

**Pull request merged**